src/tiledimage.js

```diff
@@ -311,6 +311,10 @@ $.extend($.TiledImage.prototype, $.EventSource.prototype, /** @lends OpenSeadrag
             this._updateViewport();
             this._midDraw = false;
         }
+        // Images with opacity 0 should not need to be drawn in future. this._needsDraw = false is set in this._updateViewport() for other images.
+        if (this.opacity === 0) {
```

Comment:
Minor point, but let's just make this an `else` from the if statement above; feels cleaner to me.